app/components/Header/HeaderLinks.jsx

```diff
@@ -15,7 +15,7 @@ import headerLinksStyle from './headerLinksStyle';
 
 class HeaderLinks extends React.Component {
```

Comment:
Some linting here. We still have issues with linter not going through all files... I'll fix it later